CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 # Changelog
 
-## [v0.5.0](https://github.com/mvgijssel/arel_toolkit/tree/v0.4.10) (2023-04)
+## [v0.6.0](https://github.com/mvgijssel/arel_toolkit/tree/v0.6.0) (2023-04)
```

Comment:
The changelog is generated with `rake changelog` whenever a new version is released, no need to do this manually!

Reply:
https://github.com/mvgijssel/arel_toolkit/pull/210#discussion_r1168521150